src/ekore/anomalous_dimensions/unpolarized/space_like/as4/fhmruvv_approximations/__init__.py

```diff
@@ -0,0 +1,55 @@
+"""The |N3LO| Altarelli-Parisi splitting kernels, from Falcioni, Herzog, Moch, Ruijl, Ueda, Vermaseren, Vogt approximations."""
```

Comment:
```suggestion
"""The |FHMRUVV| |N3LO| Altarelli-Parisi splitting kernels approximations."""
```

Reply:
Maybe also add to the docs here a sentence like "Remember that the authors follow Pegasus convention and so there is an additional global minus sign with respect to our conventions"

Reply:
yes and no - I think it is the usual Pegasus sign: it uses the bare Mellin transform of splitting functions, but we use anomalous dimensions defined as (- Mellin transform of splitting functions) [as the various MVV papers do], no?

Reply:
okay let's add a sentence.